Topics/Tech_Stacks/TinyMCE.md

```diff
@@ -0,0 +1,39 @@
+## TinyMCE API with ReactJS
+
+### What is TinyMCE API
+TinyMCE API is an free API that allows you to add an WYSIWYG(what you see is what you get) HTML editor on your website. This API essentially makes you textbox interactive for instance users can enter text, image and more in the textbox. This API is especially helpful if your app is heavy on editing or involves a lot of text changes from the user.
+
+### Relevant links:
+Github: https://github.com/tinymce/tinymce
+Official website: https://www.tiny.cloud/
+Documentation: https://www.tiny.cloud/docs/
+
+### Steps to integrate TinyMCE API to your React app:
+* Prerequisites: Node.js and npm
+below is an article about getting them if you don't have them installed
+https://radixweb.com/blog/installing-npm-and-nodejs-on-windows-and-mac
+
+1) Run the following commands:
+- if you already have an React App:
+    cd into your React App, and run  ```$ npm install --save @tinymce/tinymce-react``` in terminal.
+- if you don't have an React App:
+    create app: ```$ npx create-react-app tinymce-react-demo```
+    cd into directory: ```$ cd tinymce-react-demo```
+    install: ```$ npm install --save @tinymce/tinymce-react```
+
+2) Now that you have your App and TinyMCE installed, open App.jsx (or your main file) and add in the editor by importing from tinyMCE and calling it in your main function.
+```
+import { useRef } from 'react';
+import {Editor} from '@tinymce/tinymce-react';
+function App(){
+    return <>
+    <Editor apiKey='yourapikey' onInit={(evt, editor)=>editorRef.current=editor}/>
+    </>
+}
+
+export default App;
+```
+
+3) Lastly, you may notice an warning on your webpage if you try running it. This is because we still need to get the API key. To get the key, sign up through https://www.tiny.cloud/auth/signup/ and on your homepage, select React (or other if you want to integrate this API with other techstacks like Vue.js). Scroll down to find the API key, and paste it in the apiKey section in your code.
```

Comment:
It could also be beneficial to mention what exactly the warning users would see if they do not have an API key. Maybe add a picture or a description of the warning message.